lambda_functions/review_service/update_review.py

```diff
@@ -41,6 +41,17 @@ def update_review(event, context):
     except:
         return helper.get_text_response(404, "No review found", event)
 
+    # Save qualitative_comment
+    if 'qualitative_comment' in body:
```

Comment:
Is there any special reason, why the property is called "*qualitative*_comment" instead of simply "comment" like everywhere else?

Reply:
By everywhere else I mean the models, handlers, ... not the other properties